packages/backend/.gitignore

```diff
@@ -0,0 +1,8 @@
+build
+
+/dist
+.env
+.env.development
+.env.production
```

Comment:
I belive your intention was to not include the .env s in this PR, since i see an attempt tor eference them in the gitignore?
also, I belive we can just use one env for now? 
Once the auth pr gets in, maybe you can include the rapyd_api there? am handling the environment there with zod validator, parsing it and exposing it in the ap p as "env". You could make use of it.